src/styles/pdf/PdfVariables.ts

```diff
@@ -0,0 +1,28 @@
+export const PAGE_PADDING = 20;
```

Comment:
Let's rename the file to `pdfVariables` or `pdfStyleTokens`.
Only components should have filenames starting with uppercase.